lib/elixir/lib/kernel/special_forms.ex

```diff
@@ -316,16 +316,24 @@ defmodule Kernel.SpecialForms do
   defmacro unquote(:<<>>)(args), do: error!([args])
 
   @doc """
-  Defines a remote call or an alias.
+  Defines a remote call, a call to an anonymous function, or an alias.
 
   The dot (`.`) in Elixir can be used for remote calls:
 
       iex> String.downcase("FOO")
       "foo"
 
   In this example above, we have used `.` to invoke `downcase` in the
-  `String` alias, passing "FOO" as argument. We can also use the dot
-  for creating aliases:
+  `String` alias, passing "FOO" as argument.
```

Comment:
While we are here we should fix this sentence since it is incorrect. We cannot invoke things on aliases since it's a compile time construct in the AST only. I propose: "In this example above, we have used `.` to invoke `downcase` on the `String` module, passing "FOO" as argument.".